.gitignore

```diff
@@ -33,3 +33,4 @@ tools/conf.ini
 tools/custom_provision
 api/bots/john/assets/var/database.db
 static/locale/en
+.mypy_cache/
```

Comment:
Oh interesting. That'd certainly explain why nobody had added it before!

I'll drop this commit, then.